example/lib/subscription_handlers/browser_registration_subscriptions.dart

```diff
@@ -0,0 +1,21 @@
+import 'dart:async';
+
+import 'package:onegini/events/browser_event.dart';
+import 'package:onegini/events/onewelcome_events.dart';
+import 'package:onegini/onegini.dart';
+import 'package:onegini/user_client.dart';
+import 'package:onegini_example/ow_broadcast_helper.dart';
+
+// Event Subscriptions related to the creation of Pin
+class BrowserRegistrationSubscriptions {
```

Comment:
Do these really need to be classes? Can't we just have them as normal functions and non-static?